coalib/results/Diff.py

```diff
@@ -523,12 +542,12 @@ def modify_line(self, line_nr, replacement):
         if linediff.change is not False and linediff.change[1] != replacement:
             if len(replacement) == len(linediff.change[1]) == 1:
                 raise ConflictError('Cannot merge the given line changes.')
-
```

Comment:
minor and up to you: wanna keep this blank line? I think it improves readability^^